### PR TITLE
Draft page for the 2022 MFEM workshop

### PR DESCRIPTION
**Pull request by tzanio**
TODO

- [x] Add a news item on main page
- [x] Update workshop.md, e.g. bring in backgrounds
- [x] Update simulation contest form (create a new form?)
- [x] Update registration form (feedback from last time)

### CODE REVIEW

File: src/news.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 <img width=130px, style="margin:-10px"> | |
 ------------ | -----------------------------------------------------------------
+**Aug 18, 2021** | *The second [MFEM Community Workshop](workshop.md) will take place on October 25th, 2022.*
```
Comment (acfisher) [inline src/news.md:5]: Aug 18, 2022

File: src/workshop.md [hunk 2]
```diff
@@ -25,71 +18,101 @@ directions for the project.
 
 ### Registration
 
-Registration closed on October 18th.
+If you plan to attend, [please register no later than **October 18th**](https://forms.gle/voPNCQWwXDHZ96uQ6).
```
Comment (acfisher) [inline src/workshop.md:21]: We should probably ask folks that want to present to have their registration in by an earlier date, maybe Oct1?

Comment (tzanio) [reply src/workshop.md:21]: Yep, we can do that

**Comment by tzanio** [comment]
@pazner, should we make another Google form for the simulation images contest, or reuse the current one?

**Comment by pazner** [comment]
> @pazner, should we make another Google form for the simulation images contest, or reuse the current one?

I don't see an issue reusing the current one.

**Comment by tzanio** [comment]
This is ready for merging. Can I get some approvals?

**Review by hauten — APPROVED** [review]

**Review by psocratis — APPROVED** [review]

File: src/index.md [hunk 3]
```diff
@@ -87,10 +87,10 @@ See also our [Gallery](gallery.md), [Publications](publications.md), [Videos](vi
 
 Date         | Message
 ------------ | -----------------------------------------------------------------
+Aug 18, 2022 | [MFEM Community Workshop](workshop.md) in October.
```
Comment (pazner) [inline src/index.md:90]:
```suggestion
Aug 18, 2022 | [MFEM Community Workshop](workshop.md) in October 2022.
```

Somewhat redundant, but maybe clearer?

**Review by pazner — APPROVED** [review]
Looks great!

**Pull request merged**